cmd/agent/subcommands/run/command.go

```diff
@@ -183,6 +183,22 @@ import (
 	ddruntime "github.com/DataDog/datadog-agent/pkg/runtime"
 )
 
+var (
+	agentStarted = pkgTelemetry.NewCounter(
```

Comment:
Can we not use the global `pkgTelemetry` instead of the telemetry component? It would require defining the metrics in the `run` function. Since this metrcis are only used in that function I believe there is no issue on doing that + we do not relay in the global `pkgTelemetry`